The LOOP: ... messages appear to be errors, but in reality they are
simply debug messages that indicate there is one or more dependency
loop within the system.

Due to Poky packages owning all of the directories they use, a large
number of depedency loops may be present.  So disabling this message
seems to make the most sense.

Signed-off-by: mark.hatle@windriver.com

diff -ur rpm-5.1.10.orig/lib/depends.c rpm-5.1.10/lib/depends.c
--- rpm-5.1.10.orig/lib/depends.c	2010-09-29 12:23:48.307048065 -0500
+++ rpm-5.1.10/lib/depends.c	2010-09-29 12:33:23.306022259 -0500
@@ -2044,9 +2044,11 @@
 	 * Attempt to unravel a dependency loop by eliminating Requires's.
 	 */
 	if (zap) {
+#if 0
 	    rpmlog(msglvl,
 			_("removing %s \"%s\" from tsort relations.\n"),
 			(rpmteNEVRA(p) ?  rpmteNEVRA(p) : "???"), dp);
+#endif
 	    rpmteTSI(p)->tsi_count--;
 	    if (tsi_prev) tsi_prev->tsi_next = tsi->tsi_next;
 	    tsi->tsi_next = NULL;
@@ -2676,10 +2678,12 @@
 		/* Unchain predecessor loop. */
 		rpmteTSI(p)->tsi_chain = NULL;
 
+#if 0
 		if (!printed) {
 		    rpmlog(msglvl, _("LOOP:\n"));
 		    printed = 1;
 		}
+#endif
 
 		/* Find (and destroy if co-requisite) "q <- p" relation. */
 		dp = zapRelation(q, p, 1, &nzaps, msglvl);
